lib/tasks/migrations/create_2021_ivl_pacakges.rake

```diff
@@ -7,7 +7,7 @@ namespace :import do
     hbx = HbxProfile.current_hbx
 
     # Second lowest cost silver plan
-    slcs_products = BenefitMarkets::Products::Product.where(hios_id: "94506DC0390005-01")
+    slcs_products = BenefitMarkets::Products::Product.where(hios_id: "33653ME0050002-01")
```

Comment:
@eidarus, go into `disable_shop_base` and take the original file `lib/tasks/migrations/create_2021_ivl_pacakges.rake` in `disable_shop_base` and rename it `create_2020_ivl_packages_DC.rake` (or something similar, make sure we get the spelling of `packages` right this time).

Then, go into `disable_shop_base` and do something like `git checkout origin/me_demo lib/tasks/migrations/create_2021_ivl_pacakges.rake` to get the version we have changed in the branch, and rename it something like `create_2021_ivl_packages_maine.rake`. 

Make sure both of the tasks run without errors in `disable_shop_base`.

Then there should be no differences with this file on this branch. It's just a rake task- so it's not a problem if its in the `disable_shop_base` branch.